server/src/project/project.service.ts

```diff
@@ -283,8 +284,50 @@ export class ProjectService {
   constructor(
     private readonly dynamicsWebApi: OdataService,
     private readonly config: ConfigService,
+    private readonly carto: CartoService,
   ) {}
 
+  async getBblsGeoJSON(bbls) {
+    let collectedBBLs = bbls;
+
+    if (collectedBBLs === null) return null;
+
+    collectedBBLs = collectedBBLs.filter(bbl => bbl !== null);
+
+    const SQL = `
+        SELECT
+          ST_AsGeoJSON(ST_Multi(ST_Union(the_geom))) AS polygons
+        FROM mappluto
+        WHERE bbl IN (${collectedBBLs.join(',')})
+        GROUP BY version
+      `;
+
+    const cartoResponse = await this.carto.fetchCarto(SQL, 'json', 'post');
+
+    if (cartoResponse.length === 0) {
+      return {
+        polygons: null,
+      };
+    }
+
+    // return first object in carto response, carto.sql always return an array
+    return cartoResponse[0];
```

Comment:
Is there a way we can move some of this munging stuff into the carto service? It seems like the responsibility of that service. could also request `geojson` — I think that return a less awkward structure?

Reply:
Good idea -- will look into it

Reply:
Hmm actually moving `[0]` into the service would break the abstraction/utility of the carto service, since I think it's possible there's more than one result for certain SQL queries.